commons/commands.py

```diff
@@ -524,6 +524,7 @@
 K8S_POD_INTERACTIVE_CMD = "kubectl exec -it {} -c cortx-hax -- {}"
 K8S_DATA_POD_SERVICE_STATUS = "consul kv get -recurse | grep s3 | grep name"
 K8S_CONSUL_UPDATE_CMD = 'kubectl exec -it {} -c {} -- {}'
+K8S_APPLY_FILE = "kubectl apply -f {}"
```

Comment:
Added this K8S_APPLY_YAML_CONFIG in https://github.com/Seagate/cortx-test/pull/1732/files

Reply:
Yes, will make necessary changes